frontend/src/metabase/components/Popover/TippyPopover.tsx

```diff
@@ -86,9 +84,6 @@ function TippyPopover({
   const shouldShowContent = mounted && content != null;
   const isControlled = props.visible != null;
 
-  const theme = useMantineTheme();
-  const { zIndex = DEFAULT_Z_INDEX } = theme.other.popover ?? {};
```

Comment:
Because we backported `--mb-popover-z-index` in this PR, we can safely remove this and use the CSS variables instead. Note that we still use the `DEFAULT_Z_INDEX` value (4) as a default zIndex instead of 200.